django_declarative_apis/machinery/tasks.py

```diff
@@ -26,8 +26,10 @@
     import cid.locals
```

Comment:
Since we use it in testing, shoudl we add `django-cid` to our `optional-dependencies`?

Reply:
we're actually mocking it in testing, so it's not necessary.  What's the rule for putting things in `optional-dependencies`?  It doesn't seem to simplify installing things with pip

Reply:
Oh- maybe you overload the necessary method in testing.  I'm trying to think about where, if anywhere, `django-cid` should go in our listed dependencies in `pyproject.toml`.  Maybe nowhere, and (ideally) we would just document that DDA supports it somewhere (README or docs).  That doesn't need to be in this PR.